saw-core/src/Verifier/SAW/FiniteValue.hs

```diff
@@ -121,11 +124,16 @@ instance Show FirstOrderValue where
       FOVIntMod _ i -> shows i
       FOVWord _ x -> shows x
       FOVVec _ vs -> showString "[" . commaSep (map shows vs) . showString "]"
-      FOVArray{}  -> shows $ firstOrderTypeOf fv
+      FOVArray _kty d vs ->
+        let vs' = map showEntry $ Map.toAscList vs
+            d' = showEntry ("<default>", d)
+        in
+        showString "[" . commaSep (vs' ++ [d']) . showString "]"
       FOVTuple vs -> showString "(" . commaSep (map shows vs) . showString ")"
       FOVRec vm   -> showString "{" . commaSep (map showField (Map.assocs vm)) . showString "}"
     where
       commaSep ss = foldr (.) id (intersperse (showString ",") ss)
+      showEntry (k, v) = shows k . showString ": " . shows v
```

Comment:
Would you be opposed to using something like `:=` here instead of `:`? (The latter makes me think of a typing judgment.